FormsExtension/index.html

```diff
@@ -15,7 +15,7 @@
          gtag('config', 'G-L4MQFNK7VK');
    </script>
 <body>
-  	<div><p>Datalogics offers the Forms Extension, a software module that allows applications built using the Adobe PDF Library to work with PDF AcroForm and XFA forms documents. The extension works with the Windows 32 and 64-bit Adobe PDF Library platforms and is available for separate purchase.</p>
+  <div><p>Datalogics offers the Forms Extension, a software module that allows applications built using the Adobe PDF Library to work with PDF AcroForm and XFA forms documents. The extension works with the Windows x64 and Linux x86_64 Adobe PDF Library platforms and is available for separate purchase.</p>
```

Comment:
I think we need to explain that we ADDed Linux x86_64 in the 18.x version - not just change the platforms here.